bodhi/client/__init__.py

```diff
@@ -144,16 +144,6 @@ def updates():
 @click.option('--file', help='A text file containing all the update details')
 @url_option
 def new(user, password, url, **kwargs):
-    """
-    Create a new update.
```

Comment:
We do want to keep the descriptions - it's just the Args we don't want. The descriptions are useful in the help text.